test/integration/model/optimistic_locking.test.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const Support = require(__dirname + '/../support')
+  , DataTypes = require(__dirname + '/../../../lib/data-types')
+  , chai = require('chai')
+  , expect = chai.expect;
+
+describe(Support.getTestDialectTeaser('Model'), function() {
+  describe('optimistic locking', function () {
+    var Account;
```

Comment:
I know that sometimes `this` is used, but in this test file it's only used _once_, so everything else can be arrows 😄